src/pages/SubmissionDetails.js

```diff
@@ -81,8 +81,11 @@ export const SubmissionDetails = ({ submissionId }) => {
       const runsAnnotatedWithDurations = _.map(r => _.merge(r, { duration: getDuration(r.state, r.submission_date, r.last_modified_timestamp, isRunInTerminalState) }), runs)
       setRunsData(runsAnnotatedWithDurations)
 
-      // only refresh if there are Runs in non-terminal state
-      if (_.some(({ state }) => !isRunInTerminalState(state), runs)) {
+      const loadedRunSetData = await loadRunSetData(signal)
+      setRunSetData(loadedRunSetData)
+
+      // only refresh if there are Run Sets/Runs in non-terminal state
```

Comment:
small nit: this can now be updated to `only refresh if the Run Set is in non-terminal state`